### PR TITLE
Frontend: hide create buttons from unauthorized users

### LINKED ISSUE #681: Unauthorized user should NOT be able to see "add new tool/organization" buttons
image

### PR DESCRIPTION
**Pull request by sanyavertolet**
This PR closes #681 

### What's done:
- [x] Hid "Add new tested tool" button from unauthorized users
- [x] Hid "Add new organization" button from unauthorized users

### CODE REVIEW

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt [hunk 1]
```diff
@@ -18,13 +19,22 @@ import react.dom.*
 import react.table.columns
 
 import kotlinx.html.ButtonType
+import kotlinx.html.hidden
+
+/**
+ * `Props` retrieved from router
+ */
+@Suppress("MISSING_KDOC_CLASS_ELEMENTS")
+external interface CreationViewProps : PropsWithChildren {
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt:28]:
```suggestion
external interface CreationViewProps : Props {
```
I think we had `PropsWithChildren` here by mistake, it's not necessary

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt [hunk 2]
```diff
@@ -83,7 +93,9 @@ class CollectionView : AbstractView<PropsWithChildren, State>(false) {
         "LongMethod",
     )
     override fun RBuilder.render() {
+        // these buttons will be hidden while local testing so you need to ping localhost:8080/#/creation manually
```
[the remark this answers is not in the document]
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt:96]: I do not think that there is any need to spend our valuable time on mocking that. These views are available by the direct link

**Review by akuleshov7 — APPROVED** [review]

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt:96]: Ok will remove it

**Comment by akuleshov7** [comment]
https://blog.gradle.org/gradle-flaky-test-retry-plugin

**Review by petertrr — APPROVED** [review]

**Pull request merged**